source/_integrations/group.markdown

```diff
@@ -32,6 +32,8 @@ group:
     entities:
       - device_tracker.dad_smith
       - device_tracker.mom_smith
+  # Example quick group
+  dining_room: light.chandelier, switch.hutch
```

Comment:
We should add this. The configuration variables say that `entities:` is required. This will lead to confusion.

Reply:
I'm not a big fan of the comma-separated list. It is a "hack" for Home Assistant, and not a YAML feature. For that reason, I'm against using that in examples.